lib/core/src/Cardano/Pool/DB/Model.hs

```diff
@@ -430,6 +444,16 @@ mListHeaders k
     | k > 0 = reverse . take k <$> get #blockHeaders
     | otherwise = reverse <$> get #blockHeaders
 
+mReadSettings
+    :: ModelOp Settings
+mReadSettings = get #settings
+
+{-# HLINT ignore mPutSettings "Use const" #-}
```

Comment:
Why? (do not answer to me, but please add a comment in code explaining your rationale for disabling this hint).